### ISSUE TITLE
Shell Theme: Add Circle of Friends Logo to App Grid Icon

### ISSUE THREAD

**Issue by HenryRabelo** [ubuntu/yaru#2172]
Hello, I just want to start by saying that I greatly appreciate all the effort the Yaru team has put in making this an excellent theme for the Ubuntu Desktop experience.

  Having said that, I can't help but feel that an important part of the previous desktop experience is missing. One of the defining features that helped consolidate the branding of the Ubuntu Desktop, besides the dock, was the separate logo that resided in said dock.

  It was something that could indicate, at a glace, what OS the computer was running. This feature was so recognizable that every simplified concept of the desktop included it and Canonical continued its inclusion on the Unity 8 desktop *and* phone concepts.

  As such, I want to suggest the reimplementation of said logo, as I think it would greatly benefit the desktop experience. The Ubuntu Dock is something unique to the Ubuntu Desktop, and it can be played around freely, as a defining characteristic of the experience, therefore the perfect place for the logo.

I have made a concept of the desired implementation, based on the Unity 8 Redesigned Shell Theme, and it involves just a few modifications to the current Shell Theme, as follows:

#dashtodockContainer .show-apps .overview-icon, .check_flag_start {

	background-image: url("ubuntu-logo-icon.svg") !important;
	background-size: contain;
	background-color: transparent !important;
	color: transparent;

}

#dashtodockContainer.extended .show-apps, .check_flag_end {

	background-color: #E95420 !important;
	background-gradient-direction: vertical;
	background-gradient-start: transparent;
	background-gradient-end: #F7713B;
	box-shadow:inset 0 0px 1px 1px rgba(255,255,255,0.1);

}

  Of course, this can also be implemented without the gradient, with just the background-color active. Here's a screenshot of the results:

![Screenshot](https://user-images.githubusercontent.com/53522727/80628828-55bd0f80-8a28-11ea-9734-8d352dafb7da.png)

  Again, I greatly appreciate all the effort the Yaru team has put into this theme and I hope this suggestion will be taken into consideration.

**Comment by clobrano** [comment]
Hey @HenryRabelo, thank you for the feedback and for taking time to contribute with the mockup, we all really appreciate these efforts.

We discussed this a lot in the past (see for example #2006), and we ended up deciding to keep the upstream's grid.

This is not to say that I don't want, or like, the proposal, but the current grid is probably a better fit for the functionality of the button.

Thanks again
